<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GestionarAvionesDialog</class>
 <widget class="QDialog" name="GestionarAvionesDialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>769</width>
    <height>351</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>769</width>
    <height>351</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>50000</width>
    <height>50000</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Gesti칩n de aviones</string>
  </property>
  <property name="windowIcon">
   <iconset resource="recursos.qrc">
    <normaloff>:/iconos/images/iconowp_8Vn_icon.ico</normaloff>:/iconos/images/iconowp_8Vn_icon.ico</iconset>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QScrollArea" name="scrollArea">
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="scrollAreaWidgetContents">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>176</width>
          <height>292</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_7">
         <item>
          <widget class="QLabel" name="avionesDefinidosLabel">
           <property name="font">
            <font>
             <pointsize>10</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Aviones definidos</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QVBoxLayout" name="layoutAvionesDefinidos"/>
         </item>
         <item>
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QGroupBox" name="informacionGrupo">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>Informaci칩n general</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLabel" name="empresaLabel">
              <property name="text">
               <string>Empresa</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="empresaTextEdit"/>
            </item>
            <item>
             <widget class="QLabel" name="modeloLabel">
              <property name="text">
               <string>Modelo</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="modeloTextEdit"/>
            </item>
            <item>
             <widget class="QLabel" name="versionLabel">
              <property name="text">
               <string>Version</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="versionTextEdit"/>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QGroupBox" name="parametrosGrupo">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="title">
             <string>Par치metros del avi칩n</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_4">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_2">
               <item>
                <widget class="QLabel" name="maxPitchLabel">
                 <property name="minimumSize">
                  <size>
                   <width>84</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>84</width>
                   <height>16777215</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>Max. Pitch</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="maxPitchtextEdit">
                 <property name="minimumSize">
                  <size>
                   <width>120</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>120</width>
                   <height>16777215</height>
                  </size>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="grados1Label">
                 <property name="text">
                  <string>grados</string>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="horizontalSpacer_2">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item>
                <widget class="QLabel" name="minPitchLabel">
                 <property name="minimumSize">
                  <size>
                   <width>55</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>55</width>
                   <height>16777215</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>Min. Pitch</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="minPitchTextEdit">
                 <property name="minimumSize">
                  <size>
                   <width>120</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>120</width>
                   <height>16777215</height>
                  </size>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="grados2Label">
                 <property name="text">
                  <string>grados</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_3">
               <item>
                <widget class="QLabel" name="velocidadLabel">
                 <property name="minimumSize">
                  <size>
                   <width>84</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>84</width>
                   <height>16777215</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>Velocidad crucero</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="velocidadTextEdit">
                 <property name="minimumSize">
                  <size>
                   <width>120</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>120</width>
                   <height>16777215</height>
                  </size>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="msLabel">
                 <property name="text">
                  <string>m/seg.</string>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="horizontalSpacer">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_5">
               <item>
                <widget class="QLabel" name="recorridoLabel">
                 <property name="minimumSize">
                  <size>
                   <width>84</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>84</width>
                   <height>16777215</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>Recorrido circuit</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="recorridoTextEdit">
                 <property name="minimumSize">
                  <size>
                   <width>120</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>120</width>
                   <height>16777215</height>
                  </size>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="metros1Label">
                 <property name="text">
                  <string>metros</string>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="horizontalSpacer_4">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item>
                <widget class="QLabel" name="baseLabel">
                 <property name="minimumSize">
                  <size>
                   <width>55</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>55</width>
                   <height>16777215</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>Base circuit</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="baseTextEdit">
                 <property name="maximumSize">
                  <size>
                   <width>120</width>
                   <height>16777215</height>
                  </size>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="metros2Label">
                 <property name="text">
                  <string>metros</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_6">
               <item>
                <widget class="QLabel" name="headerFicheroLabel">
                 <property name="minimumSize">
                  <size>
                   <width>84</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>Fichero Header</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="headerPathText">
                 <property name="enabled">
                  <bool>false</bool>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="headerBoton">
                 <property name="text">
                  <string>Seleccionar</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_7">
               <item>
                <widget class="QLabel" name="footerFicheroLabel">
                 <property name="minimumSize">
                  <size>
                   <width>84</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>Fichero Footer</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="footerPathText">
                 <property name="enabled">
                  <bool>false</bool>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="footerBoton">
                 <property name="text">
                  <string>Seleccionar</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="layoutBotoneraEdicion">
            <item>
             <spacer name="horizontalSpacer_3">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="exportarBoton">
              <property name="text">
               <string>Exportar</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="cancelarBoton">
              <property name="text">
               <string>Cancelar</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="aceptarBoton">
              <property name="text">
               <string>Aceptar</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="eliminarBoton">
              <property name="text">
               <string>Eliminar</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="editarBoton">
              <property name="text">
               <string>Editar</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="crearBoton">
              <property name="text">
               <string>Crear</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="botonera">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="recursos.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>botonera</sender>
   <signal>accepted()</signal>
   <receiver>GestionarAvionesDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>botonera</sender>
   <signal>rejected()</signal>
   <receiver>GestionarAvionesDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
